goss-testing/scripts/check_k8s_loadbalancers.sh

```diff
@@ -53,23 +54,61 @@ function get_default_net_from_sls() {
       echo >&2 "error: failed to get Networks from SLS"
       exit 1
   fi
-
-  defaultRoute=$(echo "${NETWORKSJSON}" | jq -r --arg n "BICAN" '.[] | select(.Name == $n) | .ExtraProperties | .SystemDefaultRoute')
-  echo "$defaultRoute"
+  echo "$NETWORKSJSON"
 }
 
-default_bican_net=$(get_default_net_from_sls)
+function check_chn_config_from_sls() {
+
+  defaultRoute=$(echo "$1" | jq -r --arg n "BICAN" '.[] | select(.Name == $n) | .ExtraProperties | .SystemDefaultRoute')
 
-# Check to see if the CHN is configured by checking the BICAN toggle in SLS
-chn_configured=0
-if [ "$default_bican_net" == "CHN" ]; then
+  # Check to see if the CHN is configured by checking the BICAN toggle in SLS
+  chn_configured=0
+  if [ "$defaultRoute" == "CHN" ]; then
     chn_configured=1
-else
+  else
     chn_configured=0
-fi
+  fi
+
+  echo "$chn_configured"
+}
+
+function check_cmn_config_from_sls() {
+
+  cmn_config=$(echo "$1" | jq -r --arg n 'customer-management-static' '.[].ExtraProperties.Subnets[]| select(.MetalLBPoolName == $n)')
```

Comment:
This will work for now since we expect we will always have a static pool for the CMN.  You could also just check that the CMN network exists.
```
echo "$1" | jq -r --arg n "CMN" '.[] | select(.Name == $n)'
```